test/Microsoft.Azure.ServiceBus.UnitTests/SenderReceiverTests.cs

```diff
@@ -459,5 +459,16 @@ public async Task ClientsUseGlobalConnectionCloseFirstClientSecoundClientShouldS
                 await receiver.CloseAsync().ConfigureAwait(false);
             }
         }
+
+        [Theory]
+        [InlineData(TestConstants.NonPartitionedQueueName)]
+        [DisplayTestMethodName]
+        async Task MessageSenderThrowsWhenSendingEmptyCollection(string queueName)
```

Comment:
You might want to rename this method to reflect what is actually being tested

Reply:
WIP. Also, I'm thinking the test is not good enough 🙂

Reply:
Yeah the test somehow doesnt look perfect. Why do we even need a receiver here? Of course there is no message sent as its an empty list. Its not worth spending those 3 seconds * number of build configurations to test this.

---
In reply to: [254793100](https://github.com/Azure/azure-service-bus-dotnet/pull/642#discussion_r254793100) [](ancestors = 254793100)

Reply:
Any ideas how to test it better? I'm open for suggestions.